### PR TITLE
Ephemeral gateways

### PR DESCRIPTION
**Pull request by AndrewDryga**
- [x] Fixed docker run command to mount a volume at `/etc/firezone`
- [x] Fixed systemd unit file to prope setcap, create writeable `/etc/firezone` directory, use non-root user, etc
- [x] Removed `FIREZONE_ID` from our terraform scripts

Now on Sites index we only show online gateways:
Screenshot 2023-11-15 at 18 04 12

On the Site view we also show only online ones with a link to see all:
Screenshot 2023-11-15 at 18 02 33

All can be seen on a separate page:
Screenshot 2023-11-15 at 18 02 27

Some of the functions I've added are pretty dirty hacks, we really need to implement filters from #2029 to properly implement those and remove code duplicates.

### CODE REVIEW

File: elixir/apps/web/lib/web/live/relay_groups/new_token.ex [hunk 1]
```diff
@@ -233,7 +248,18 @@ defmodule Web.RelayGroups.NewToken do
         chmod +x /usr/local/bin/firezone-relay; \\
       fi \\
     '
+    ExecStartPre=/usr/bin/mkdir -p /etc/firezone
```
Comment (AndrewDryga) [inline elixir/apps/web/lib/web/live/relay_groups/new_token.ex:268]: @conectado please LMK if you see that something is off here.

File: elixir/apps/web/assets/pnpm-lock.yaml [hunk 2]
```diff
@@ -22,8 +22,8 @@ packages:
     resolution: {integrity: sha512-P1st0aksCrn9sGZhp8GMYwBnQsbvAWsZAX44oXNNvLHGqAOcoVxmjZiohstwQ7SqKnbR47akdNi+uleWD8+g6A==}
     dev: false
 
-  /flowbite@2.0.0:
-    resolution: {integrity: sha512-gP/iC/WuznQ5XBzikhaSs4RDs49zrvoAdHbWMHSY3l7nVJX0xJz+dELIlLjh+czLdEVTMLxUjuARYYwCb5q34A==}
+  /flowbite@2.1.1:
```
Comment (AndrewDryga) [inline elixir/apps/web/assets/pnpm-lock.yaml:25]: Bumped it hoping to fix #2657

Comment (jamilbk) [reply elixir/apps/web/assets/pnpm-lock.yaml:25]: If that doesn't work you may find this helpful: https://github.com/firezone/firezone/blob/main/website/src/components/DocsSidebar/index.tsx#L12

Comment (jamilbk) [reply elixir/apps/web/assets/pnpm-lock.yaml:25]: Also happy to assign that issue to me to take a look at

File: elixir/apps/web/lib/web/live/sites/gateways/index.ex [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+              <div>
+                <p>
+                  Gateways serve as a termination point for all VPN connections within a site, they must be deployed
+                  to your infrastructure to enable VPN connectivity.
+                </p>
+                <p>
+                  If site has more than one gateway deployed,
+                  connections will be load balanced between them. Because of this, all gateways within a site must
+                  be able to access all site resources.
+                </p>
```
Comment (jamilbk) [inline elixir/apps/web/lib/web/live/sites/gateways/index.ex]: Minor suggestion for simplification

```suggestion
                <p>
                  Deploy gateways to terminate connections to your site's resources. All gateways deployed within a site must be able to reach all its resources.
                </p>
```

File: elixir/apps/web/lib/web/live/sites/new_token.ex [hunk 4]
```diff
@@ -195,9 +201,13 @@ defmodule Web.Sites.NewToken do
     [Unit]
     Description=Firezone Gateway
     After=network.target
+    Documentation=https://www.firezone.dev/docs
```
Comment (jamilbk) [inline elixir/apps/web/lib/web/live/sites/new_token.ex]:
```suggestion
    Documentation=https://www.firezone.dev/kb
```

**Review by jamilbk — CHANGES_REQUESTED** [review]
Definitely a step in the right direction! URL fix and a helptext suggestion

File: elixir/apps/web/lib/web/live/sites/gateways/index.ex (hunk 3 above)
Comment (AndrewDryga) [reply elixir/apps/web/lib/web/live/sites/gateways/index.ex]: This was a copy-paste from another page, I've fixed it in both places.

File: elixir/apps/web/assets/pnpm-lock.yaml (hunk 2 above)
Comment (AndrewDryga) [reply elixir/apps/web/assets/pnpm-lock.yaml:25]: @jamilbk even if we manually re-initialize Flowbyte data toggles they will lose the state of the active element (so you was on step 3 of Systemd guide and then you see docker again), I think the best solution here is not to use Flowbyte JS and just implement active tab tracking with LiveView.

Comment (AndrewDryga) [reply elixir/apps/web/assets/pnpm-lock.yaml:25]: Just verified it using phx-hooks, re-init doesn't work and I don't see any useful functions in the docs: https://flowbite.com/docs/components/tabs/

Comment (jamilbk) [reply elixir/apps/web/assets/pnpm-lock.yaml:25]: gotcha, I've been deep into flowbite for the website stuff, so I can take a look at the issue when I get a chance

**Review by jamilbk — APPROVED** [review]

**Pull request merged**